rules/rbe_repo.bzl

```diff
@@ -456,8 +502,7 @@ _rbe_autoconfig = repository_rule(
         ),
         "digest": attr.string(
             mandatory = True,
```

Comment:
Add a example to make it explict that `sha256` needs to be included in the string.